### ISSUE TITLE
Dual Display Split Screen doesn't work with Steam Remote Play

### ISSUE THREAD

**Issue by Zortabre** [henpemaz/RemixMods#15]
When the game is split into 2 windows, the non-host player sees a screen that says "Input is temporarily disabled while the host is busy"

**Comment by ECX-0** [comment]
Yeah this is an issue with steam that I don't believe is possible to be fixed unless maybe the host plays as player 2, I suggest just using parsec instead because steam remote play isn't meant for that
